docs/whats-new/staging-release-notes.md

```diff
@@ -2,6 +2,19 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Staging release 21 December 2022
+Frontend version:  | Backend version: 
+
+**Changes** 
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Manage application role assignments now using TechBiz portal</summary>
+
+Tenants who have onboarded to TechBiz, need to use [TechBiz portal](https://portal.stg.techbiz.suite.gov.sg/) to manage the role assignments for it.
```

Comment:
to manage the role assignments for 'it' <-- 'it' here refers to? or shall we just say 'to manage application role assignments'

Also i checked with Lijia, they will only enforce this 4 Jan, so perhaps we add in front 'From 4 January 2022 onwards, ...'

Reply:
done